### PR TITLE
Pull request for WAZO-2422-switchboard-fallback-noanswer

### PR DESCRIPTION
**Pull request by sduthil**
## switchboards: extract list of queued calls to ari_helpers

Why:

* Will be needed by no-answer timeout

## switchboard fallbacks: replace wazo-confd with channel vars

Why:

* Avoid dependency on wazo-confd

## tests switchboard: improve assertion for timeout fallback

Why:

* If something fails, channel may get hungup for the wrong reason

### CODE REVIEW

**Comment by sduthil** [comment]
recheck

**Comment by sduthil** [comment]
recheck

File: wazo_calld/plugins/switchboards/services.py [hunk 1]
```diff
@@ -81,9 +90,88 @@ def new_queued_call(self, tenant_uuid, switchboard_uuid, channel_id):
...
+            switchboard_uuid,
+            channel_id,
+            noanswer_timeout,
+        )
+        timer = threading.Timer(
+            noanswer_timeout,
+            self.on_queued_call_noanswer_timeout,
+            args=(tenant_uuid, switchboard_uuid, channel_id),
+        )
+        timer.start()
```
[the remark this answers is not in the document]
Comment (pc-m) [reply wazo_calld/plugins/switchboards/services.py:121]: 4, 2, 1, 3, 5

[the remark this answers is not in the document]
Comment (sduthil) [reply wazo_calld/plugins/switchboards/services.py:121]: OK, we're on the same page. Are you OK with merging this PR to get the whole feature to master, while I work on adding `
`asyncio`?

[the remark this answers is not in the document]
Comment (pc-m) [reply wazo_calld/plugins/switchboards/services.py:121]: Yes

**Review by pc-m — APPROVED** [review]

**Pull request merged**